flog/__init__.py

```diff
@@ -0,0 +1,21 @@
+from flask import Flask
+from configs.conf import database, secret_key, username, password, debug, upload_folder, allowed_extensions
+from services.logger.logger import log_error
+
+app = Flask(__name__, static_url_path = '/static')
+
+from flog import controllers, models
+
+app.config.update(DATABASE = database,
+                  SECRET_KEY = secret_key,
+                  USERNAME = username,
+                  PASSWORD = password,
+                  DEBUG = debug,
+                  UPLOAD_FOLDER = upload_folder,
+                  ALLOWED_EXTENSIONS = allowed_extensions
```

Comment:
For what reason do you use this app configuration? ...cause I saw you use config and get all parameters from there